README.md

```diff
@@ -203,9 +203,9 @@ level list: `DEBUG`, `INFO`, `WARN`, `ERROR`, `NONE`
 
 mock home directory
 
-### mm.restore
+### mm.restore()
 
-restore all mock data, e.g. `afterEach(mm.restore)`
+restore all mock data, e.g. `afterEach(mm.restore())`
```

Comment:
I see!

I removed this changes, make it only apply to the the title. In previous, mm.restore looks like a property to me. To make it consistent with `mm.home(homePath)` or mm.consoleLevel(level)`, changed mm.restore to mm.restore().